src/main/webapp/js/index.js

```diff
@@ -15,12 +15,12 @@
 
 /** Fetches receipts from the server and adds them to the DOM. */
 async function searchReceipts() {
-  const label = document.getElementById('search-input').value;
-  const response = await fetch(`/search-receipts?label=${label}`);
+  const categories = document.getElementById('category-input').value;
+  const response = await fetch(`/search-receipts?categories=${categories}`);
```

Comment:
Similar comment to your other PR: I would consider using [URLSearchParams](https://developers.google.com/web/updates/2016/01/urlsearchparams) to build the parameter list or use encodeURIComponent to make sure the user input is properly encoded (e.g., will the above won't handle if the input includes a space like "burger king").

Reply:
It handles spaces already